demo/unity/PorcupineDemo.cs

```diff
@@ -8,30 +7,57 @@
 using Pv.Unity;
 
 public class PorcupineDemo : MonoBehaviour {
-    
-    public Texture[] _imgs = new Texture[7];
-    List<string> _keywords = new List<string> { "americano", "grasshopper", "grapefruit", "computer", "blueberry", "bumblebee", "porcupine" };
-    
+
+    private const string ACCESS_KEY = "${YOUR_ACCESS_KEY_HERE}"; // AccessKey obtained from Picovoice Console (https://picovoice.ai/console/)
+
+    static List<Porcupine.BuiltInKeyword> _keywords = Enum.GetValues(typeof(Porcupine.BuiltInKeyword)).Cast<Porcupine.BuiltInKeyword>().ToList();
+    public Texture[] _imgs;
+
     Button _startButton;
     RawImage _outputImg;
+    Text _textField;
     Color _alphaSubtract = new Color(0, 0, 0, 0.008f);
 
     private bool _isProcessing;
     PorcupineManager _porcupineManager;
-    
+    private bool isError = false;
+
     void Start() 
     {
         _startButton = gameObject.GetComponentInChildren<Button>();
         _startButton.onClick.AddListener(ToggleProcessing);        
         _outputImg = gameObject.GetComponentInChildren<RawImage>();
+        _textField = gameObject.GetComponentInChildren<Text>();
+        _keywords.Remove(Porcupine.BuiltInKeyword.HEY_GOOGLE);
+        FillKeywords();
 
         try
         {
-            _porcupineManager = PorcupineManager.FromKeywords(_keywords, OnWakeWordDetected);            
+            _porcupineManager = PorcupineManager.FromBuiltInKeywords(ACCESS_KEY, _keywords, OnWakeWordDetected, processErrorCallback: ErrorCallback);            
+        }
+        catch (PorcupineInvalidArgumentException ex)
+        {
+            SetError($"{ex.Message}\n. Make sure your access key '{ACCESS_KEY}' is a valid access key.");
```

Comment:
`SetError($"{ex.Message}\nMake sure your access key...`
Looks weird with the period and space after the newline